docs/how_to_guides/miscellaneous/how_to_add_and_test_a_new_sqlalchemydataset_class.rst

```diff
@@ -0,0 +1,242 @@
+            )
+
+    * **In build_test_backends_list, add a variable and if clause**
+
+        .. code-block:: python
+
+            no_presto = metafunc.config.getoption("--no-presto")
+            if not no_presto:
+                presto_conn_str = "presto://presto@localhost/memory/test_ci"
+                try:
+                    engine = sa.create_engine(presto_conn_str)
+                    conn = engine.connect()
+                except (ImportError, sa.exc.SQLAlchemyError):
+                    raise ImportError(
+                        "presto tests are requested, but unable to connect to the presto database at "
+                        f"'{presto_conn_str}'"
+                    )
+                test_backends += ["presto"]
+
+    * **In the sa fixture method, add your test_backend to the list of backends**
+
+        .. code-block:: python
+
+            if "postgresql" not in test_backends and "sqlite" not in test_backends and "presto" not in test_backends:
+
+
+    * **For each of the test datasets, add a schema entry for the dialect.**
+
+        Examples:
+
+            * In ``numeric_high_card_dataset`` : ``"presto": {"norm_0_1": "DOUBLE"},``
+            * In ``datetime_dataset`` : ``"presto": {"datetime": "TIMESTAMP"},``
+            * In ``dataset_sample_data`` : ``"presto": {"infinities": "DOUBLE", "nulls": "DOUBLE", "naturals": "DOUBLE"},``
+
+
+4. Make changes to tests/test_utils.py
+######################################
+
+    * **Add a try-except clause to import dialect-specific types and map them to generic types.**
+
+        .. code-block:: python
+
+            try:
+                import sqlalchemy.types as sqltypes
+                from pyhive.sqlalchemy_presto import presto as prestotypes
+                from pyhive.sqlalchemy_presto import PrestoDialect as prestodialect
+
+                PRESTO_TYPES = {
+                    "VARCHAR": sqltypes.VARCHAR,
+                    "TEXT": sqltypes.VARCHAR,
+                    "CHAR": sqltypes.CHAR,
+                    "DOUBLE": prestotypes.DOUBLE,
+                    "INTEGER": sqltypes.INTEGER,
+                    "SMALLINT": sqltypes.SMALLINT,
+                    "BIGINT": sqltypes.BIGINT,
+                    "DATETIME": sqltypes.TIMESTAMP,
+                    "TIMESTAMP": sqltypes.TIMESTAMP,
+                    "DATE": sqltypes.DATE,
+                    "FLOAT": prestotypes.DOUBLE,
+                    "BOOLEAN": prestotypes.BOOLEAN,
+                }
+            except ImportError:
+                PRESTO_TYPES = {}
+
+    * **In get_dataset, add an elif clause to connect to data, .** Note: some SQL backends require a schema. If so, this is also the right place to create a test schema, usually called ``test_ci``.
+
+        .. code-block:: python
+
+            elif dataset_type == "presto":
+                from sqlalchemy import create_engine
+
+                engine = create_engine("presto://presto@localhost/memory/test_ci", echo=True)
```

Comment:
I've set the echo to False now (that's what we probably want when we merge the tests)

Reply:
```suggestion
                engine = create_engine("presto://presto@localhost/memory/test_ci", echo=False)
```